v1-preparations/LIST_OF_STABLE_RESOURCES_FOR_V1.md

```diff
@@ -1,107 +1,108 @@
-    * [snowflake_stream_on_external_table](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/stream_on_external_table)
-    * snowflake_stream_on_directory_table
-    * snowflake_stream_on_view
-    * [snowflake_streams](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/data-sources/streams) (datasource)
+* Stream
+    * [snowflake_stream_on_table](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/resources/stream_on_table)
+    * [snowflake_stream_on_external_table](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/resources/stream_on_external_table)
+    * [snowflake_stream_on_directory_table](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/resources/stream_on_directory_table)
+    * [snowflake_stream_on_view](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/resources/stream_on_view)
+    * [snowflake_streams](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/data-sources/streams) (datasource)
 * Streamlit
-    * [snowflake_streamlit](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/streamlit)
-    * [snowflake_streamlits](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/data-sources/streamlits) (datasource)
+    * [snowflake_streamlit](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/resources/streamlit)
+    * [snowflake_streamlits](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/data-sources/streamlits) (datasource)
 * Table (in progress)
-    * [snowflake_table](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/table)
-    * [snowflake_tables](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/data-sources/tables) (datasource)
+    * [snowflake_table](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/resources/table)
+    * [snowflake_tables](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/data-sources/tables) (datasource)
 * Tag (in progress)
-    * [snowflake_tag](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/tag)
-    * [snowflake_tag_association](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/resources/tag_association)
-    * [snowflake_tags](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.97.0/docs/data-sources/tags) (datasource)
+    * [snowflake_tag](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/resources/tag)
+    * [snowflake_tag_association](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/resources/tag_association)
+    * [snowflake_tags](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/0.98.0/docs/data-sources/tags) (datasource)
```

Comment:
Why not just `latest`? Are we going to update this file with each release?

Reply:
because we have no control over the latest; also, after the V1 release, the latest will point to the 1.x.x versions, and I think this should point to the exact release candidates.